api/bash/migrate-repos-in-org.sh

```diff
@@ -0,0 +1,391 @@
+            # Migrating repos and updating teams #
+            ######################################
+            echo "-----------------------------------------------------"
+            echo "Migrating Repo:[$new_repo] to:[$MASTER_ORG/$new_repo]"
+            #####################################
+            # Call GitHub =API to transfer repo #
+            #####################################
+            curl -s --request POST \
+               --url $GITHUB_URL/repos/$ORIG_ORG/$new_repo/transfer \
+               --header "authorization: Bearer $GITHUB_TOKEN" \
+               --header 'content-type: application/json' \
+               --header 'application/vnd.github.nightshade-preview+json' \
+               --data  \"{\"new_owner\": \"$MASTER_ORG\", \"team_ids\": [ $TEAM_IDS ]}\"
+
+            ########################
+            # Check for any errors #
+            ########################
+            if [ $? -ne 0 ]; then
+               echo "Error! Failed to migrate repo"
+               ((ERROR_COUNT++))
+            fi
+            
+            ###########################
+            # Update Team permissions #
+            ###########################
+            UpdateTeamPermission $new_repo
+         fi
+      else
+         # Debug loop to print results
+         echo "DEBUG: Would have moved:[$new_repo] to:[$MASTER_ORG/$new_repo]"
+      fi
+   done
+}
+################################################################################
+#### Function Footer ###########################################################
+Footer()
+{
+   ####################
+   # Print the footer #
+   ####################
+   echo "-----------------------------------------------------"
+   echo "the script has completed"
+   exit $ERROR_COUNT
+}
+################################################################################
+#### Function Header ###########################################################
+Header()
+{
+   #####################
+   # Print Header Info #
+   #####################
+   echo "-----------------------------------------------------"
+   echo "-----------------------------------------------------"
+   echo "----- Migrate Repos from user Org to Master Org -----"
+   echo "-----------------------------------------------------"
+   echo "-----------------------------------------------------"
+   echo ""
+   echo "Migrating All Repositories from Org:[$ORIG_ORG]"
+   echo "Moving all Repositories to Org:[$MASTER_ORG]"
+   ##############
+   # Debug info #
+   ##############
+   if [ $DEBUG -eq 1 ]; then
+      echo "Running in DEBUG mode! Will only report Repositories that will be migrated"
+   else
+      echo "Running in Execute mode! Will migrate all repositories"
+   fi
+   #############
+   # Team Info #
+   #############
+   if [ $UPDATE_TEAMS -eq 1 ]; then
+      echo "Updating Repositories teams when migrating"
+   else
+      echo "No teams will be set during the migration process"
```

Comment:
`set` => `assigned` perhaps to stick with language of the endpoint documentation in that you are "assigning" a repo to a team?